### ISSUE TITLE
center Parameter is giving Error

### ISSUE THREAD

**Issue by sameersb** [jperelli/osm-static-maps#6]
Thanks for the great library!

But when center parameter is passed to the url, it gives error!
Please check examples
http://osm-static-maps.herokuapp.com/?width=512&height=512&center='0,0'
http://osm-static-maps.herokuapp.com/?width=512&height=512&center=0,0

Regards

**Comment by n1k0** [comment]
Trying another approach : http://osm-static-maps.herokuapp.com/?center=48.8976,2.2574&zoom=15

Now the server crashes with 

```
Error: TypeError: 2.2574.split is not a function
    at <anonymous>:16:51
```

**Comment by jperelli** [comment]
Hi @sameersb, you're welcome. Hi @n1k0. I released a new version 3.4.1 that should fix this issue. 

Also @n1k0 note that the point is Lon Lat, so probably you need to switch the numbers http://osm-static-maps.herokuapp.com/?center=2.2574,48.8976&zoom=15

Feel free to reopen if you see the issue again. Thanks for using the package and reporting issues. Also share it!
Regards

**Comment by n1k0** [comment]
@jperelli thanks, works like a charm now :+1:
